Library/Formula/domoticz.rb

```diff
@@ -0,0 +1,58 @@
+class DomoticzDownloadStrategy < SubversionDownloadStrategy
+  def stage
+    cp_r "#{cached_location}/.", Dir.pwd
+  end
+end
+
+class Domoticz < Formula
+  homepage "http://www.domoticz.com/"
+  head "http://svn.code.sf.net/p/domoticz/code/trunk/", :using => DomoticzDownloadStrategy
+
+  depends_on "cmake" => :build
+  depends_on "boost" => :build
+  depends_on "openssl"
+  depends_on "libusb"
+  depends_on "libusb-compat"
+  depends_on "homebrew/dupes/zlib"
```

Comment:
Turns out it can use the system version. Installing zlib was in the domoticz installation instructions so I followed it. Good call!